src/include/containers/Map.h

```diff
@@ -0,0 +1,173 @@
+#ifndef CUHKSZ_CONTAINERS_MAP
+#define CUHKSZ_CONTAINERS_MAP
+
+#include <map>
+#include <utility>
+
+namespace cuhksz {
+
+template <typename KeyType, typename ValueType>
+class Map {
+public:
+	typedef std::pair<const KeyType, ValueType> StdValueType;
+	Map();
+	Map(const Map& other);
+	Map( std::initializer_list<StdValueType> init );
+
+	~Map();
+
+	Map& operator =(Map& map2);
+
+	ValueType& get(const KeyType& key);
+	const ValueType& get(const KeyType& key) const;
```

Comment:
well....it seems key check is not suitable at here, I also think causing an error when user want to set a non-exists key is not user-friendly. I'll delete keychecking.